src/test/scripts/functions/caching/BufferpoolLeak.dml

```diff
@@ -22,7 +22,7 @@
 X = rand(rows=$1, cols=$2, min=1, max=10);
 for(i in 1:500) {
   #print("executed iteration "+i)
-  [m1,m2] = mice(as.frame(X), matrix(0,1,ncol(X)),3,3, FALSE)
+  [m1,m2] = mice(X), matrix(0,1,ncol(X)), 3, FALSE)
```

Comment:
It seems like the parenthesis is incorrect here.
Does this not crash the system?

Reply:
> It seems like the parenthesis is incorrect here.
> Does this not crash the system?

I fixed the call, surprisingly it was not crashing the test on git. After fixing the call now the cache is leaning and BufferpoolleakTest is failing.